src/block-locking.jsx

```diff
@@ -66,7 +66,7 @@ export function setupBlockLocking( governanceRules ) {
 
 				return <>
 					<Disabled>
-						<div style={ { opacity: 0.6, 'background-color': '#eee', border: '2px dashed #999' } }>
+						<div style={ { opacity: 0.6, 'backgroundColor': '#eee', border: '2px dashed #999' } }>
```

Comment:
@smithjw1 Thank you for jumping on here really fast! Tiny nit, we can remove the quotes around `'backgroundColor'` now to match `opacity`:

```suggestion
						<div style={ { opacity: 0.6, backgroundColor: '#eee', border: '2px dashed #999' } }>
```

Feel free to make that change (or not) and merge.